configs/anvil/config.20170926.FCT2.A_WCYCL1850S.ne30_oECv3.anvil

```diff
@@ -117,7 +117,6 @@ mldSubdirectory = MLD
 ninoSubdirectory = Nino
 mhtSubdirectory = MHT
 
-
```

Comment:
I think we want to keep 2 blank lines between each section so it's easier to find section.  @gstreletz did a lot of work to put in this formatting.

Reply:
Oops, sorry.  I thought this was config.default.  Ignore my comment above.

Reply:
I only removed it because it wasn't consistent in the config example cases: in most cases there weren't blank lines, so I thought of just removing this one.
Instead, config.defaults has the blank lines between each section.